container-compose.yml

```diff
@@ -10,6 +10,12 @@ services:
       - db
       - redis
       - yarn
+    healthcheck:
+      test: ["CMD-SHELL", "curl -f http://localhost:3000/ || exit 1"]
```

Comment:
- Does this need `CMD-SHELL`? Can it use `CMD` like `redis-cli` does below?
- Can we get away without changing the return code here? `curl(1)` is already pretty good about returning a myriad of non-zero error codes in the event of failure, and I'm not aware of a reason to coalesce them all to `1`. See for example:

```sh
(woods) ~  » curl foo.bar.example.com; echo $?
curl: (6) Could not resolve host: foo.bar.example.com
6
```

Reply:
The same :). I will switch to a CMD.